app/views/users/_form.html.erb

```diff
@@ -3,20 +3,7 @@
 <% if f.error_messages != "" %><div class="alert alert-danger"><%= f.error_messages %></div><% end %>
```

Comment:
For the modals to work, we need to supply the url in the form. So we created a new _form2 partial for the user creation view and used existing _form partial for the edit user view. So removing these redundant code now.